pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests that a device gives the same output of the default device."""
+# pylint: disable=no-self-use
+from pennylane import numpy as np
+import pytest
+from flaky import flaky
+from pennylane.templates.layers import RandomLayers
```

Comment:
These codefactor warnings could be resolved by running `isort` on this file. `isort` can be installed with `pip` and it sorts the imports in the expected order.